### ISSUE TITLE
Error: AttributeError: module 'tensorflow.python.keras.utils.generic_utils' has no attribute 'populate_dict_with_module_objects'

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/gcaer#3]
When running:

```
/home/richel/.local/share/r-miniconda/envs/r-reticulate/bin/python ~/.local/share/gcaer/gcae_v1_0/run_gcae.py
```

I get

```
Traceback (most recent call last):
  File "/home/richel/.local/share/gcaer/gcae_v1_0/run_gcae.py", line 31, in <module>
    import tensorflow as tf
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/__init__.py", line 41, in <module>
    from tensorflow.python.tools import module_util as _module_util
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 84, in <module>
    from tensorflow.python import keras
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/__init__.py", line 27, in <module>
    from tensorflow.python.keras import models
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/models.py", line 24, in <module>
    from tensorflow.python.keras import metrics as metrics_module
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/metrics.py", line 37, in <module>
    from tensorflow.python.keras.engine import base_layer
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/engine/base_layer.py", line 51, in <module>
    from tensorflow.python.keras import initializers
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/initializers/__init__.py", line 127, in <module>
    populate_deserializable_objects()
  File "/home/richel/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/keras/initializers/__init__.py", line 85, in populate_deserializable_objects
    generic_utils.populate_dict_with_module_objects(
AttributeError: module 'tensorflow.python.keras.utils.generic_utils' has no attribute 'populate_dict_with_module_objects'
```

**Comment by richelbilderbeek** [comment]
So, from [here](https://stackoverflow.com/a/67667006):

> change `from keras import models` to `from tensorflow.keras import models`
> this solved the problem for me with tensorflow 2.5.0

or, from [here](https://stackoverflow.com/a/61347413):

> I had the same problem, and I have successfully solved this issue with downgrading tensorflow version to 2.1.0.
>
> `pip install tensorflow==2.1.0`

**Comment by richelbilderbeek** [comment]
I feel this fixed for now.
